pkg/uconn/results.go

```diff
@@ -50,10 +56,13 @@ func LongConnResults(res *resources.Resources, thresh int, limit int, noLimit bo
 			"dst":              1,
 			"dst_network_uuid": 1,
 			"dst_network_name": 1,
+			"count":            1,
+			"tbytes":           1,
+			"tdur":             1,
 			"tuples":           bson.M{"$slice": []interface{}{"$tuples", 5}},
 			"open":             1,
 		}},
-		{"$sort": bson.M{"maxdur": -1}},
+		{"$sort": bson.M{"tdur": -1}},
```

Comment:
Would you mind adding maxdur as a secondary sort here? That way if someone is looking at an older dataset it obviously won't have anything in the tdur column but at least the results will still be sorted by maxdur.

Reply:
Added